src/interfaces/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@bcrs-shared-components/interfaces",
-    "version": "1.0.9",
+    "version": "1.1.0",
```

Comment:
You don't do this. Lerna does.

Reply:
oh nice, will correct, thanks!

Reply:
If these steps needs tweaking, feel free to update them!
https://github.com/bcgov/bcrs-shared-components/blob/master/README.md